parsing/lexer.mll

```diff
@@ -512,6 +520,14 @@ rule token = parse
             { INFIXOP3(Lexing.lexeme lexbuf) }
   | '#' (symbolchar | '#') +
             { HASHOP(Lexing.lexeme lexbuf) }
+  | '`' anycase identchar *
+      { BACKQUOTEIDENT (get_quote_name lexbuf) }
+  | '`' anycase_latin1 identchar_latin1 *
+      { warn_latin1 lexbuf; BACKQUOTEIDENT (Lexing.lexeme lexbuf) }
```

Comment:
Why are you dropping the backquote with `get_quote_name` in the `anycase` case but not `anycase_latin1`? (Same with QUOTEIDENT below). Note that you don't need to write `String.sub 1 ...` for this, you should be able to use

```
| '`' ((anycase identchar*) as ident)

Reply:
This was a mistake due to multiple iterations. Fixed.